### PR TITLE
GH-968 Fix AtmospheresToPascals and PascalsToAtmospheres methods

### PR DESCRIPTION
**Pull request by pictos**
### Description of Change ###

Correct the calc inside the method. 

### Bugs Fixed ###

- Related to issue #968 

Provide links to issues here. Ensure that a GitHub issue was created for your feature or bug fix before sending PR.

### Behavioral Changes ###

Now the methods AtmospheresToPascals and PascalsToAtmospheres returns the correct values.

### PR Checklist ###

- [x] Has tests (if omitted, state reason in description)
- [x] Has samples (if omitted, state reason in description)
- [x] Rebased on top of master at time of PR
- [x] Changes adhere to coding standard
- [x] Updated documentation ([see walkthrough](https://github.com/xamarin/Essentials/wiki/Documenting-your-code-with-mdoc))

### CODE REVIEW

File: Xamarin.Essentials/Xamarin.Essentials.csproj [hunk 1]
```diff
@@ -21,7 +21,7 @@
     <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
     <RepositoryUrl>https://github.com/xamarin/Essentials</RepositoryUrl>
     <PackageReleaseNotes>See: https://aka.ms/xamarinessentials-releasenotes</PackageReleaseNotes>
-    <LangVersion>7.3</LangVersion>
+    <LangVersion>8.0</LangVersion>
```
Comment (Redth) [inline Xamarin.Essentials/Xamarin.Essentials.csproj]: Can you please remove this language change from the PR, then I think it's good to go, thanks!

Comment (pictos) [reply Xamarin.Essentials/Xamarin.Essentials.csproj]: sure.

**Comment by pictos** [comment]
@Redth I already did the changes, but it looks like the CI doesn't like me. Am I doing something wrong?

**Comment by jamesmontemagno** [comment]
CI should now be working again, validated the formulas. Good catch!

**Pull request merged**